tiledb/sm/misc/utils.h

```diff
@@ -46,6 +46,14 @@
 namespace tiledb {
 namespace sm {
 
+/** Class for generic status exceptions. */
+class GenericException : public StatusException {
```

Comment:
We don't want to use this. Calling a `StatusException` generic requires subverting the requirement that we track origin.

Reply:
Removed